apps/execute-actions-react/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import mondaySdk from "monday-sdk-js";
+const monday = mondaySdk();
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { board: {} };
+  }
+
+  componentDidMount() {
+    monday.listen("context", this.getContext);
+  }
+
+  getContext = res => {
+    this.setState({ context: res.data });
+
+    // normalize the data as it can be retrieved from board view or widget that
+    // supports multiple boards
+    let boardIds;
+    if (this.state.context.boardIds) {
```

Comment:
`const currentBoardIds = this.state.context.boardIds`

Reply:
thats what i'm checking thought. it can be in. boardIds or boardId